packages/SwingSet/misc-tools/perc100.py

```diff
@@ -0,0 +1,25 @@
+import csv
+import sys
+import itertools
+import statistics
+
+SIZE = 1000
+
+print("first_cranknum,deliver_median,gc_p10,gc_p50,gc_p90")
```

Comment:
What is this script expecting standard input to contain, and what would produce it?

Reply:
Huh, good question, I forget. I'll delete it.